app/models/work.rb

```diff
@@ -0,0 +1,48 @@
+class Work < ApplicationRecord
+  has_many :votes
+  has_many :users, :through => :votes
+
+  validates :category, presence: true 
+  validates :title, presence: true 
+  validates :title, uniqueness: { case_sensitive: false} 
+  validates :creator, presence: true 
+  validates :publication_year, presence: true 
+  validates :publication_year, numericality: { only_integer: true }
+  validates :description, presence: true 
+
+  def self.sort_album_works
+    # puts "LOG: #{Vote.group(:work_id).count}"
+    return Work.where(category: "album").left_joins(:votes).group(:id).order('COUNT(votes.id) DESC')
+  end
+
+  def self.sort_book_works
+    return Work.where(category: "book").left_joins(:votes).group(:id).order('COUNT(votes.id) DESC')
+  end
+
+  def self.sort_movie_works
+    return Work.where(category: "movie").left_joins(:votes).group(:id).order('COUNT(votes.id) DESC')
+  end
+
```

Comment:
Notice that these methods are pretty much the same.  You could instead use one method with a parameter.